openstack_commands/network.sh

```diff
@@ -37,6 +37,18 @@ openstack subnet create \
           --dns-nameserver 8.8.8.8 \
           --dhcp subnet-external
 
+# NESE storage network
+openstack network create --provider-network-type vlan --provider-segment 211 --provider-physical-network datacentre  --share nese-storage
+openstack subnet create \
+          --network nese-storage \
+          --subnet-range 10.0.120.0/22  \
+          --ip-version 4 \
+          --allocation-pool start=10.0.121.0,end=10.0.123.254 \
```

Comment:
wouldn't the first host be `10.0.121.1` for this subnet which is already allocated to the gateway, so start should probably be `10.0.121.2`. 

Also, do you think it makes sense to reserve some static addresses on this network, so maybe we want to start with `10.0.120.10`?

Reply:
10.0.120.1 is the first host, I reserved that whole /24 for static IPs, everything else is dhcp (10.0.120.1-10.0.120.255 is not DHCP allocated)

Reply:
oh yeah, you are right. I missed that it's 121 and not 120.